src/client/rpc.rs

```diff
@@ -153,6 +154,8 @@ impl RpcClient {
         }
 
         if res.is_invalid() {
+            counter!("rpc_fork_choice_updated_v3_invalid_payload").increment(1);
```

Comment:
```suggestion
            counter!("rpc_fork_choice_updated_v3_invalid_payload_{self.payload_source}").increment(1);
```

not sure about syntax but this is my idea